repos/fdbt-netex-output/src/data/auroradb.ts

```diff
@@ -23,7 +23,7 @@ export const getAuroraDBClient = (): Pool => {
             password: awsParamStore.getParameterSync('fdbt-rds-netex-output-password', { region: 'eu-west-2' }).Value,
             database: 'fdbt',
             waitForConnections: true,
-            connectionLimit: 10,
+            connectionLimit: 1,
```

Comment:
Is this saying on prod, the connection limit is `1`?

Aren't databases able to handle 1000s of connections at the same time? I'm confused, can you shed some light on this, please?

Reply:
This means each Lambda will use 1 connection but we can have up to 40 instances running at the same time which will mean the connections are shared out between all the lambdas. The DB size we use only supports 90 connections